src/Okta.Idx.Sdk/IIdxClient.cs

```diff
@@ -192,5 +236,13 @@ public interface IIdxClient : IOktaClient
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>The identity providers response.</returns>
         Task<IdentityProvidersResponse> GetIdentityProvidersAsync(IIdxContext idxContext, CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Poll the status of an Okta Verify enrollment.
+        /// </summary>
+        /// <param name="idxContext">The IDX context.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <returns>The authentication response.</returns>
+        Task<PollResponse> PollEnrollmentStatusAsync(IIdxContext idxContext, CancellationToken cancellationToken = default);
```

Comment:
I suggest renaming this to `PollAuthenticatorEnrollmentStatusAsync` to be consistent with the other Poll method.